inc/user.class.php

```diff
@@ -2413,9 +2413,16 @@ function showMyForm($target, $ID) {
 
          if (!GLPI_DEMO_MODE) {
             $langrand = mt_rand();
-            echo "<td><label for='dropdown_language$rand'>" . __('Language') . "</label></td><td>";
-            // Use session variable because field in table may be null if same of the global config
-            Dropdown::showLanguages("language", ['value' => $this->fields["language"], 'rand' => $langrand]);
+            echo "<td><label for='dropdown_language$langrand'>" . __('Language') . "</label></td><td>";
+            // Language is stored as null in DB if value is same as the global config.
+            Dropdown::showLanguages(
+               "language",
+               [
+                  'display_emptychoice' => true,
+                  'rand'                => $langrand,
+                  'value'               => $this->fields["language"],
```

Comment:
Guess it would be better to check if `$this->fields['language']` is null and take glpi global preference instead; empty choice does not reflect the current configuration.
Or maybe use sthing like "From GLPI global configuration" as empty label.

Reply:
I changed it to use the global config if user lang is null.